### PR TITLE
Document OP 34 (Request Last Messages)

### PR DESCRIPTION
**Pull request by aamiaa**
image

image

Only checks the first 100 ids:

image
image

### CODE REVIEW

File: pages/topics/gateway-events.mdx [hunk 1]
```diff
@@ -333,6 +333,31 @@ Used to request a private channel's pre-existing call data, created before the G
 }
 ```
 
+#### Request Last Messages
+
+Used to request the last messages (indicated by the `last_message_id` field) from channels. Responds with a single [Last Messages](#last-messages) event with up to 100 messages.
```
Comment (dolfies) [inline pages/topics/gateway-events.mdx:338]:
```suggestion
Used to request the last messages (indicated by the [`last_message_id` field](/resources/channel#channel-object)) from channels. Fires a [Last Messages](#last-messages) Gateway event with up to 100 messages that match the request.
```

File: pages/topics/gateway-events.mdx [hunk 2]
```diff
@@ -333,6 +333,31 @@ Used to request a private channel's pre-existing call data, created before the G
...
+
+###### Request Last Messages Structure
+
+| Field           | Type             | Description                                          |
+|-----------------|------------------|------------------------------------------------------|
+| guild_id        | snowflake        | The ID of the guild                                  |
+| channel_ids ^1^ | array[snowflake] | The IDs of the channels to request last messages for |
+
+^1^ You can only provide up to 100 channel IDs. If you send more than that, the excessive ones will be ignored.
```
Comment (dolfies) [inline pages/topics/gateway-events.mdx:347]:
```suggestion
| channel_ids ^1^ | array[snowflake] | The IDs of the channels to request last messages for (max 100) |
```
I feel like this is a weird discord quirk that doesn't really need specific outlining, not sure tho <:bleh:1047549509745524777>

**Review by dolfies — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a couple of nitpicks.

File: pages/topics/gateway-events.mdx (hunk 2 above)
Comment (aamiaa) [reply pages/topics/gateway-events.mdx:347]: okay <a:cheeks:1055631376319266937>

**Review by dolfies — APPROVED** [review]

**Pull request merged**